includes/core/service/README.md

```diff
@@ -0,0 +1,21 @@
+# WooCommerce Payments Services
+
+## Purpose
+
+Expose WooCommerce Payments functionality through Services. Extensions should use these integration points provided in the `WCPay\Core` namespace, and should not use any methods from other classes, which are considered internal to WCPay. 
+
+In this way, extensions will be able to use services in a reliable, and compatible manner. 
+
+## Services available
+
+- [`WC_Payments_Customer_Service_API`](customer-service.md)
+
+## Adding New Services
+
+- Add the new service within `includes\core\service`.
+- Service should be in `WCPay\Core` namespace.
+- Add functions in your service that act as proxies to the internal service methods, or consider how they might change in the future.
+- Add tests in `tests\unit\core\services`.
```

Comment:
```suggestion
- Add tests in `tests\unit\core\service`.
```

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/5608/commits/b372b5247d0ea5b1074323a0946b4f8057fd4dbe 👍